client/src/linkManager.js

```diff
@@ -75,6 +94,110 @@ export class linkManager extends LuigiClientBase {
     this.navigate(path, 0, true, modalSettings || {});
   }
 
+  /**
+   * Opens a view in a splitted view. You can specify the split view's title and size. If you don't specify the title, it is the node label. If there is no node label, the title remains empty.  The default size of the split view is `l`, which means 80%. You can also use `m` (60%) and `s` (40%) to set the split view size. Optionally, use it in combination with any of the navigation functions.
+   * @memberof linkManager
+   * @param {string} path navigation path
+   * @param {Object} splitViewSettings opens a view in a split view. Use these settings to configure the split view's behaviour
+   * @param {string} splitViewSettings.title split view title. By default, it is the node label. If there is no label, it is left empty
+   * @param {number} [splitViewSettings.size=40] size of the split view in percent
+   * @example
+   * LuigiClient.linkManager().openAsSplitView('projects/pr1/logs', {title: 'Logs', size: 40});
+   */
+  openAsSplitView(path, splitViewSettings) {
+    this.navigate(path, 0, true, undefined, splitViewSettings || {});
+    const sendSplitViewAction = (action, data) => {
+      window.parent.postMessage({
```

Comment:
Please note this will need to be updated once [_Post message target handling in Luigi Client_](https://github.com/SAP/luigi/pull/651) is merged